deploy/crds/operator.tigera.io_installations_crd.yaml

```diff
@@ -35,6 +35,14 @@ spec:
             description: Specification of the desired state for the Calico or Calico
               Enterprise installation.
             properties:
+              bpfDataplaneMode:
+                description: 'BPFDataplaneMode configures whether the BPF data plane
+                  is enabled anda in what form. Default: Disabled'
+                enum:
+                - Disabled
+                - Enabled
+                - EnabledDSR
```

Comment:
Not sure if DSR should be combined in here or left as a separate option (or even configured through felix config).  Probably need opinions form the operator team.